docs/backends/index.md

```diff
@@ -1,51 +1,37 @@
 # Backends
 
-This document describes the classes of backends, how they work, and any details
-about each backend that are relevant to end users.
-
-For more information on a specific backend, check the next backend pages:
-
-There are currently three classes of backends that live in ibis.
-
-1. String generating backends
-1. Expression generating backends
-1. Direct execution backends
-
 ## String Generating Backends
 
-The first category of backend translates ibis expressions into strings.
-Generally speaking these backends also need to handle their own execution. They
-work by translating each node into a string, and passing the generated string
-to the database through a driver API.
+The first category of backend translate Ibis expressions into strings.
+Generally speaking these backends handle their own execution.
```

Comment:
That's not necessarily true since pyspark and datafusion handles their own execution as well but we utility a different API to talk with them. 

I'd say that these backends are databases with a SQL interface.

Reply:
I think this sentence that I've written here is ambiguous and possibly just wrong, I'll reword it.

Reply:
I'm dubious on the value of this entire file, as it's a wall of text that no one is likely to get much value out of.

Reply:
It would be useful for ibis contributors though.

Reply:
Yeah, we can keep it for now.